examples/todo-list/src/repositories/todo-list-image.repository.ts

```diff
@@ -28,9 +35,51 @@ export class TodoListImageRepository extends DefaultCrudRepository<
     protected todoListRepositoryGetter: Getter<TodoListRepository>,
   ) {
     super(TodoListImage, dataSource);
-    this.todoList = this._createBelongsToAccessorFor(
+    this.todoList = this.createBelongsToAccessorFor(
       'todoList',
       todoListRepositoryGetter,
     );
   }
+
+  async find(
+    filter?: Filter<TodoListImage>,
+    options?: Options,
+  ): Promise<TodoListImageWithRelations[]> {
+    // Prevent juggler for applying "include" filter
+    // Juggler is not aware of LB4 relations
+    const include = filter && filter.include;
+    filter = filter && Object.assign(filter, {include: undefined});
```

Comment:
Please apply the comments I posted in the docs files to the actual implementation too:

- Don't modify the original filter object
- Add a comment about a temporary implementation to all places where we are handling `include` manually, make sure it always includes a link to the GH issue tracking implementation progress